fp_output:
    formats: ["FP", "LLH"]
    type: serial
    port: "/dev/ttyUSB0"
    serial_baudrate: 115200
    rate: 200
    reconnect_delay: 5.0 # wait time in [s] until retry connection

customer_input:
    speed_topic: "/fixposition/speed"
